docs/conf.py

```diff
@@ -26,7 +26,7 @@
 ]
 
 html_theme = "rocm_docs_theme"
-html_theme_options = {"flavor": "rocm-docs-home"}
+html_theme_options = {"flavor": "rocm-blogs"}
```

Comment:
@samjwu This change is included in the final committed version, was this intentional?